### ISSUE TITLE
How does the 1-bit variant work?

### ISSUE THREAD

**Issue by ElijahHamilton** [phoboslab/qoa#47]
Hi, how does QOA work at 1-bit? Does it only store the residual at 1-bit? Or Something else?

**Comment by phoboslab** [comment]
Yes. It works similar to the other variants. Samples are divided into slices of 60. For each slice a 4-bit scalefactor and 60 1-bit residuals are stored. The 4-bit scalefactor and 1-bit residuals combined represent one of [these](https://github.com/phoboslab/qoa/blob/experimental_1bit/qoa.h#L231-L248) values.

**Comment by ElijahHamilton** [comment]
Wouldn't that give a bitrate of 8533.33333 bits/s?

**Comment by phoboslab** [comment]
For 8khz mono, yes.

I guess I misunderstood your initial question.

> Does it only store the residual at 1-bit

So, no, the scalefactor for every slice of 60 samples is also stored. That means the effective bitrate is `1.0666` bits per sample.
